src/xdocs/index.xml.vm

```diff
@@ -119,6 +119,50 @@
       </p>
     </section>
 
+    <section name="JRE and JDK">
+      <p>
+        Runtime of Checkstyle is limited only by minimal version or JRE.
+      </p>
+      <div class="wrapper">
+        <table>
+          <tr>
+            <th>Checkstyle version</th>
+            <th>JRE version</th>
+          </tr>
+          <tr>
+            <td>
+              7.x, 8.x, 9.x
+            </td>
+            <td>
+              1.8 and above
```

Comment:
I think it's better to use 8 here.

Reply:
@rnveach , @strkkk , @nmancus1 , please share your thoughts on this.
I ok with any decisions. I didn't not found consistency .
https://www.codejava.net/java-se/java-se-versions-history
And other resources think that version is 1.8 but next version is 9

Reply:
It's not a big deal, but if we do this, then why not 5 and 7 below?

Reply:
ok looks like @nmancus1  is voting for numbers without "1.".

Reply:
oracle name them without "1."
https://www.oracle.com/java/technologies/downloads/#java8
![image](https://user-images.githubusercontent.com/812984/147018105-717133a2-759a-486f-bf0b-29d07eac9ad2.png)

Reply:
I removed "1." prefix.
fixed version mapping cs6 on jre6

![image](https://user-images.githubusercontent.com/812984/147018599-b3dbb91d-eb5a-4804-920f-973c91e418b2.png)